### ISSUE TITLE
Bug with screen saver

### ISSUE THREAD

**Issue by jlozes** [Maschell/hid_to_vpad#32]
I don't know how to reproduce this bug but sometime, when I emulate a controller with my Playstation 3 controller, the screen go dark like when you don't touch the console during a moment.

I think this bug is producing when the controller is disconnected during emulation. When you reconnect it, the buttons work normaly but I think that this screen saver bug initiate at this moment.

**Comment by griest024** [comment]
Yeah the Wii U darkens the screen normally after a period of inactivity. Apparently inputs from a DS3 (mapped to pro slot 1) do not reset this timer.

**Comment by piratesephiroth** [comment]
yeah, it seems the console only cares about real button presses.
If you want to use hid_to_vpad you can just disable the burn-in-reduction feature, inside System Settings (the icon with 2 screens).

**Comment by griest024** [comment]
That's an acceptable workaround. Still, I feel like this should be fixable.

**Comment by piratesephiroth** [comment]
Yeah I bet it can be patched
